pkg/controller/node/controller_utils.go

```diff
@@ -70,6 +71,14 @@ func deletePods(kubeClient clientset.Interface, recorder record.EventRecorder, n
 			continue
 		}
 
+		// Set reason and message in the pod object.
+		if updatedPod, err := setPodTerminationReason(kubeClient, &pod, nodeName); err != nil {
```

Comment:
Maybe we should keep the error and return it after deletion.  Alternatively we can just hope we'll retry here (will we retry?)